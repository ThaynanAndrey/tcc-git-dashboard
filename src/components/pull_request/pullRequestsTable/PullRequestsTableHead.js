import React from 'react';

const PullRequestsTableHead = () => {
    return (
        <thead>
            <tr>
                <th>Pull Request</th>
                <th>Data</th>
                <th>Status</th>
                <th>Responsável</th>
                <th>Repositório</th>
                <th></th>
                <th></th>
            </tr>
        </thead>
    )
}

export default PullRequestsTableHead;