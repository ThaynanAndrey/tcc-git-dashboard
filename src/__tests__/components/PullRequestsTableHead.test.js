import React from 'react';
import { mount, configure } from "enzyme";
import toJson from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import PullRequestsTableHead from '../../components/pull_request/pullRequestsTable/PullRequestsTableHead';

describe('Test PullRequestsTableHead', () => {
  it('renders correctly a PullRequestsTableHead\'s component with correct props', () => {
    const tree = mount(
      <PullRequestsTableHead />
    );

    expect(toJson(tree)).toMatchSnapshot();
  });

  it('renders correctly a PullRequestsTableHead\'s component with correct classes', () => {
    const tree = mount(
        <PullRequestsTableHead />
      );

    expect(tree.find('thead').exists()).toBeTruthy();
    expect(tree.find('tr').exists()).toBeTruthy();
  });
})