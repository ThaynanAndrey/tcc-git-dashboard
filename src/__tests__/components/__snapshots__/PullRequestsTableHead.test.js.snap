// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PullRequestsTableHead renders correctly a PullRequestsTableHead's component with correct props 1`] = `
<PullRequestsTableHead>
  <thead>
    <tr>
      <th>
        Pull Request
      </th>
      <th>
        Data
      </th>
      <th>
        Status
      </th>
      <th>
        Responsável
      </th>
      <th>
        Repositório
      </th>
      <th />
    </tr>
  </thead>
</PullRequestsTableHead>
`;
